### PR TITLE
added note about Invalid retention policy.

### PR DESCRIPTION
**Pull request by PramodBalusu**

### CODE REVIEW

File: microsoft-365/compliance/retention.md [hunk 1]
```diff
@@ -502,6 +502,9 @@ Additional information for specific locations:
     For more information about the Preservation Hold library, see [How retention works for SharePoint and OneDrive](retention-policies-sharepoint.md#how-retention-works-for-sharepoint-and-onedrive).
 
     Because of the behavior during the grace period, if you re-enable the policy or change the location status back to on within 30 days, the policy resumes without any permanent data loss during this time.
+    
+> [!NOTE] 
+> To verify the validity of the retention policy on a SharePoint or OneDrive site and delete it if it's invalid, see [Compliance policy blocking site deletion] (https://learn.microsoft.com/sharepoint/troubleshoot/sites/compliance-policy-blocking-site-deletion).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/retention.md]:
```suggestion
> To verify the validity of the retention policy on a SharePoint or OneDrive site and delete it if it's invalid, see [Compliance policy blocking site deletion] (/sharepoint/troubleshoot/sites/compliance-policy-blocking-site-deletion).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PramodBalusu, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by cabailey** [comment]
On hold - need to discuss this with engineering

**Comment by cabailey** [comment]
Closing per email discussion

**Pull request closed without merging**